README.rst

```diff
@@ -62,3 +62,9 @@ Links
 .. _Werkzeug: https://www.palletsprojects.com/p/werkzeug/
 .. _Jinja: https://www.palletsprojects.com/p/jinja/
 .. _pip: https://pip.pypa.io/en/stable/quickstart/
+.. _LICENSE: https://github.com/pallets/flask/blob/master/LICENSE
+
+License
```

Comment:
I think this is too "noisy". Adding a link is fine (even though most people probably don't care if it's there or not), but having the section seems to be overkill.

Reply:
Thank you for your reply. I've removed the section. I understand your point